Makefile.am

```diff
@@ -81,11 +81,11 @@ CARGO_RELEASE_ARGS=--release
 endif
 
 check-local-cargo:
-	cd $(srcdir)/rust && CARGO_TARGET_DIR=$(abs_top_builddir)/target cargo test
+	CARGO_TARGET_DIR=$(abs_top_builddir)/target cargo test
```

Comment:
Do we even need to set the target dir now?

Reply:
Don't we need that for srcdir != builddir?

I mean, I'm not sure if anyone is doing that right now (the last time I tried that was a long time ago and it didn't work), but I think it'd be great if we supported it. But cool dropping it too for now until we actually make some efforts to support it. (On that subject, we should evaluate meson + cargo too as we move away from autotools.)

Reply:
> Don't we need that for srcdir != builddir?

Hmm, actually to support that, we should instead do `cd $(srcdir)` since that's where the toml is now. Anyway, for now I've just simplified it. :arrow_down: